includes/class-wc-calypso-bridge-base-admin-menu.php

```diff
@@ -0,0 +1,395 @@
+<?php
+
+/**
+ * Base Admin Menu file.
+ *
+ * @since   x.x.x
```

Comment:
Will this be auto-populated?

Reply:
We have to update it when deploying the next version.